test-files/cmd/glacial/NonContiguousRoadReviewsTest/output.osm

```diff
@@ -1,134 +1,130 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
     <bounds minlat="2.15271999999999" minlon="22.3230206655069" maxlat="2.28674376295594" maxlon="22.5485167350963"/>
```

Comment:
The change in this test output was basically a pair of roads that partially matched in the middle but didn't match at all at the ends is now completely reviewed across the whole length.  I think that's actually an improvement.  The original spirit of this test is to avoid non-contiguous reviews or reviews that pop up multiple times on the same length of road at different spots, and these changes didn't add any of those.